panel/command/__init__.py

```diff
@@ -91,7 +91,7 @@ def main(args: list[str] | None = None):
             try:
                 ret = parsed_args.invoke(parsed_args)
             except Exception as e:
-                if config.dev:
+                if hasattr(config, "dev") and config.dev:
```

Comment:
I got an error here too. I have seen this before that `config` has not attribute `dev `.

Reply:
```suggestion
                if config.autoreload:
```